src/config/cssStyles.js

```diff
@@ -0,0 +1,11 @@
+// header
+export const HEADER_SHADOW_EFFECT =
+  '0px 2px 4px -1px rgb(0 0 0 / 30%), 0px 4px 5px 0px rgb(0 0 0 / 20%), 0px 1px 10px 0px rgb(0 0 0 / 12%)';
+export const HEADER_LOGO_HEIGHT = 60;
+export const HEADER_HEIGHT = 100;
+
+// collectionCard
+export const COLLECTION_CARD_SHADOW_EFFECT =
+  '0px 2px 4px -1px rgb(0 0 0 / 30%), 0px 4px 5px 0px rgb(0 0 0 / 20%), 0px 1px 10px 0px rgb(0 0 0 / 12%)';
```

Comment:
Both shadows are actually the same? I think design wise it makes sense to have the same shadow on all elements, so I think you can have one variable only for the shadow definition.

Reply:
This two are the same, but actually some shadow effect are different. We can have a `Default shadow effect`, while the special ones can be defined seperately.